modules/nf-core/jvarkit/vcffilterjdk/meta.yml

```
@@ -0,0 +1,102 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "jvarkit_vcffilterjdk"
+description: Filtering VCF with dynamically-compiled java expressions
+keywords:
+  - vcf
+  - bcf
+  - filter
```

```suggestion
  - filter
```

Maybe stuff like `variant` etc?